ideascube/library/views.py

```diff
@@ -134,7 +134,8 @@ class SpecimenCreate(CreateView):
     def get_initial(self):
         book = get_object_or_404(Book, pk=self.kwargs['book_pk'])
 
-        return {'book': book, 'location': get_server_name()}
+        return {
+            'book': book, 'location': Configuration('server', 'site-name')}
```

Comment:
No no, the line break is OK, I was trying to prevent having line breaks everywhere we need to use the configuration in the future, as one line break is ok, but if have too much we lose a bit of readability. But OK, let's proceed :)